src/components/PeopleComponent/PeopleComponent.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { PersonLink } from '../PersonLink/PersonLink';
+import { Person } from '../../types';
+
+interface Props {
+  people: Person[]
+}
+
+export const PeopleComponent: React.FC<Props> = ({ people }) => {
+  const { slugId } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map((person) => {
+          const {
+            slug,
+            sex,
+            born,
+            died,
+            motherName,
+            fatherName,
+          } = person;
+
+          const findPersonMother = people
+            .find(item => item.name === motherName);
+
+          const findPersonFather = people
+            .find(item => item.name === fatherName);
```

Comment:
Better to prepare you people array after fetch and add `mother` and `father` to every object